### DISCUSSION TITLE
目前2.0能在condition icon 一个单元格添加多个icon了嘛 看文档并没有找到方法

### DISCUSSION THREAD

**Discussion by ornncrisol** [antvis/S2#2984] [Q&A]
![微信图片_20241120144036](https://github.com/user-attachments/assets/59b23e8f-2e77-42d8-8cf2-b14d1831c5e5)
想要在文字的左边再添加一个下钻图标

**Comment by lijinke666** [comment]
还不支持哈, 感兴趣可以参与贡献

**Comment by ornncrisol** [comment]
谢谢回复



---- 回复的原邮件 ----
| 发件人 | Jinke ***@***.***> |
| 日期 | 2024年11月21日 11:51 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***>***@***.***> |
| 主题 | Re: [antvis/S2] 目前2.0能在condition icon 一个单元格添加多个icon了嘛 看文档并没有找到方法 (Discussion #2984) |

还不支持哈, 感兴趣可以参与贡献
